jekyll/_cci2/deployment-integrations.adoc

```diff
@@ -0,0 +1,162 @@
+= Configuring Deploys
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+Deployment at CircleCI is just the same as any other job configuration; you create a job and configure it to deploy to virtually any target. This document provides an overview of the deployment process, along with best practices and optimization strategies.
+
+toc::[]
+
+[discrete]
+== Overview
+
+Once a software application has been developed and tested, it needs to be deployed and made available for its intended audience. CircleCI can be configured to deploy to any target, and can be easily configured to integrate with other services for QA/testing, feature management, deployment strategies such as blue-green or canary deployment. Quickly and easily customize your config to match your deployment strategy, whether a fully automated process or elements of manual approval are required.
+//// 
+.Deployment
+image::pipeline-to-deployment.png[Deployment]
+////
+
+== Deployment: The Basics
+
+* **Deployment Job**: To deploy your application, add a <<job-steps#jobs-overview,job>> to your `.circleci/config.yml` file and configure the job to run the steps you require. The <<deployment-examples#,deployment examples>> document covers example config for many common deployment targets.
+
+* **Deployment Parameters**: You will need to add environment variables/SSH keys to fulfill your deployment steps and provide autorisation details and parameters.
++
+These can be added to the project itself (follow links below for detailed instructions), or defined within your configuration. For an example of defining environment variables within your configuration see the non-orb Heroku example below – within the deploy job configuration we have:
++
+```
+environment:
+      HEROKU_APP: "sleepy-refuge-55486"
+```
+** Add project-level <<env-vars#setting-an-environment-variable-in-a-project,environment variables>>
+** Add project-level <<add-ssh-key#,SSH keys>> 
+
+* **Manual Approval**: If your deployment strategy requires a manual approval step, you can include a <<configuration-reference#type,hold/approve>> job within your workflow. A manual approval button will then be available from the workflows map in the CircleCI UI.
+
+== Using Orbs to Simplify Deployment
+
+https://circleci.com/docs/2.0/orb-intro/[Orbs] are available for many common deployment targets to help simplify and streamline your config – check out the full range of available orbs in the https://circleci.com/orbs/registry/[Orbs Registry]. Orbs are packages of reusable configuration, and often, for simple pipelines, simply invoking an orb, including a pre-configured job, and setting any required environment variables/SSH keys within your project will get you the results you need with no further configuration. 
```

Comment:
"Orbs are packages of reusable configuration, and often, for simple pipelines, simply invoking an orb, including a pre-configured job, and setting any required environment variables/SSH keys within your project will get you the results you need with no further configuration. "

opinion: I would suggest breaking this into two sentences to reduce the use of commas.

Reply:
ha yeah I kept reading that and thinking I need to fix it. So many commas!